src/DocBlock/Annotation.php

```diff
@@ -226,6 +226,15 @@ public function setTypes(array $types)
         $this->clearCache();
     }
 
+    public function getNormalizedTypes()
```

Comment:
please add a PHPDoc with a return type hint, should be `@return string[]` I think